EverestSplash/EverestSplash.cs

```diff
@@ -376,6 +366,40 @@ private void HandleWindow() {
                     return; // quit asap
                 }
             }
+            
+            // Update state
+            bgBloomPos += 60f*deltaTime; // 60 px per second
+            if (bgBloomPos > realBgH/2f) {
+                bgBloomPos = -realBgH/2f;
+            }
```

Comment:
```suggestion
            bgBloomPos += 60d*deltaTime; // 60 px per second
            if (bgBloomPos > realBgH/2d) {
                bgBloomPos = -realBgH/2d;
            }
```
The division will be done in a 32-bit float because of the float literal, losing precision. Use a double literal instead.